browser-extension/mv2/src/client/js/client.js

```diff
@@ -37,6 +23,19 @@
             }
           });
         }
+
+        if (!RQ.ClientUtils.isAppPage()) {
```

Comment:
The content scripts should be loaded only after the background has been notified of the page load

Reply:
In profiling, I found that all the content script listeners are attatched before the background is notified of the port connection.